stubs/aiofiles/METADATA.toml

```diff
@@ -1,6 +1,10 @@
-version = "23.1.*"
+version = "23.2.*"
 upstream_repository = "https://github.com/Tinche/aiofiles"
 
 [tool.stubtest]
 # linux and darwin are equivalent
 platforms = ["linux", "win32"]
+# Currently we are in a pickle, because aiofiles.os.__all__ is incorrect at
+# runtime when running on win32. (https://github.com/Tinche/aiofiles/pull/174)
+# This means that stubtest is either correct on Windows or all other platforms.
+skip = true
```

Comment:
Surely we don't need to skip stubtest entirely for the whole package. Can't we just create a `stubtest_allowlist_win32.txt` file in the `@tests` directory with a single entry, `aiofiles.os.__all__`?

Reply:
I didn't know we could have architecture-specific allowlists! Will change accordingly.

Reply:
One of @Avasam's innovations :)

Reply:
Embarrassingly, aiofiles already has architecture-specific allowlists ... 😳

Reply:
> One of @Avasam's innovations :)

Oh, it was @sobolevn's PR, so he deserves most of the credit! https://github.com/python/typeshed/pull/8923 Though @Avasam pushed for the change

Reply:
> Oh, it was @sobolevn's PR, so he deserves most of the credit! #8923 Though @Avasam pushed for the change

Exact :) I fixed a couple edge cases (like #9173) and updated plenty of stubs afterwards.